tools/device_generator.sh

```
@@ -45,26 +46,27 @@ done
 
 output_json=$(jq --null-input '{"devices": []}')
 
+rdma_devices=$(rdma link)
 if [[ -n $devices ]]; then
```

Shellcheck (0.10.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)